src/contexts/index.ts

```diff
@@ -39,12 +39,41 @@ export type SetState = (
   callback?: () => void,
 ) => void;
 
-export type DocStateContextProps = {
+export class BackBehaviorElements {
+  backBehaviorElements: Element[] = [];
+
+  add = (elements: Element[]): void => {
+    this.backBehaviorElements.push(...elements);
+  };
+
+  get = (): Element[] => this.backBehaviorElements;
+
+  remove = (elements: Element[]): void => {
+    if (elements.length > 0) {
+      console.log(
+        `Removing ${elements.length} elements from backBehaviorElements cur length: ${this.backBehaviorElements.length}`,
```

Comment:
I don't think we want to emit logs via `console.log` as these create performance bottleneck in the production bundle (https://reactnative.dev/docs/performance.html#using-consolelog-statements) and we have not configured our apps to remove these (which might be a performance improvement axis we explore soon).
In the future, we might want to create a logger class for Hyperview so the debugging is possible without affecting production.